doc/man3/BIO_read.pod

```diff
@@ -36,6 +38,15 @@ however; for example, BIO_gets() on a digest BIO will calculate and
 return the digest and other BIOs may not support BIO_gets() at all.
 The returned string is always NUL-terminated and the '\n' is preserved
 if present in the input data.
+On binary input there may be NUL characters within the string;
+in this case the return value (if nonnegative) may give an incorrect length.
+
+BIO_get_line() attempts to read from BIO <b> a line of data up to the next '\n'
```

Comment:
I'd somehow mention that it reads byte-by-byte. So it might not be the most effective way to read a file. Unfortunately there is no way around that.

Reply:
Good point. Done like this:
`For implementing this, unfortunately the data needs to be read byte-by-byte.`